deployments/mainnet/weth/migrations/1666906000_configurate_seed_reserves_and_rewards.ts

```diff
@@ -15,7 +15,10 @@ export default migration('1666906000_configurate_seed_reserves_and_rewards', {
     const trace = deploymentManager.tracer();
     const ethers = deploymentManager.hre.ethers;
 
-    const comptrollerV2 = await deploymentManager.fromDep('comptrollerV2', 'mainnet', 'usdc');
+    const comptrollerV2Proxy = await deploymentManager.fromDep('comptrollerV2', 'mainnet', 'usdc');
+    const comptrollerV2Impl = await deploymentManager.fromDep('comptrollerV2:implementation', 'mainnet', 'usdc');
+    const comptrollerV2 = comptrollerV2Impl.attach(comptrollerV2Proxy.address);
```

Comment:
This doesn't really make sense though, we should have it and not just sometimes, right? Seems like there's some other bug 🤔

Reply:
Which part doesn't make sense?

If you don't attach the impl contract to the proxy contract, then the contract instance won't have the impl contract functions (which we need when we call `.populateTransaction`)

Reply:
Then why doesn't `mainnet/usdc` need to do this? And also why is it working for me and 🪖 locally? https://github.com/compound-finance/comet/blob/main/deployments/relations.ts#L4-L8 spider should be crawling the impl